src/components/user-list.js

```diff
@@ -32,7 +31,8 @@ export default class UserList extends React.Component {
 
     handleClickSubmit () {
         //TODO: should be rewritten
-        let companyInfo, addressInfo = {};
+        let companyInfo, addressInfo;
```

Comment:
(code style)
It's better to assing default values right when we create variables. And we usually use the following manner
```js
let companyInfo = {};
let addressInfo = {};
```